astropy/visualization/wcsaxes/core.py

```diff
@@ -26,6 +26,14 @@
 
 VISUAL_PROPERTIES = ["facecolor", "edgecolor", "linewidth", "alpha", "linestyle"]
 
+if HAS_PIL:
+    import PIL.Image
+    if hasattr(PIL.Image, "FLIP_TOP_BOTTOM"):
```

Comment:
We should still use `minvesrion` here; it's much less of a hassle when it's not called within a loop.